sdks/java/io/elasticsearch-tests/elasticsearch-tests-5/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTest.java

```diff
@@ -185,6 +185,17 @@ public void testWritePartialUpdate() throws Exception {
     elasticsearchIOTestCommon.testWritePartialUpdate();
   }
 
+  @Test
+  public void testWritePartialUpdateWithErrors() throws Exception {
```

Comment:
rewrite this test as suggested above

Reply:
yes, it can be done like the other tests even with the index mapping overloading => I'll do it myself on another PR